.github/workflows/test.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: test
+
+on:
+  pull_request:
+  push:
+    branches:
+    - main
+    tags:
+    - '*'
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: psf/black@stable
+
+  check-file-headers:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.12
+    - name: Check header
+      run: python examples/check_header.py
+      shell: bash
+
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["ubuntu-22.04", "macos-13", "windows-2022"]
+        python_version:
+        - '3.9'
+        - '3.10'
+        - '3.11'
+        - '3.12'
+        - '3.13'
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python_version }}
+    - name: Install deps
+      run: |
+        pip install 'robotpy[rev, commands2]<2026.0.0,>=2025.0.0b3' numpy pytest
```

Comment:
So, this isn't the way we would want to do this, because what you're doing here is installing the rev library from pypi, not using the current commit.

I think it would be better to append this testing logic to the CI job (https://github.com/robotpy/robotpy-rev/blob/main/.github/workflows/dist.yml#L13) instead since it builds and installs the wheel already... or if that doesn't work, then update the template that we use to optionally have something that runs an example test script in the same way that we do test scripts (https://github.com/robotpy/build-actions/blob/v2023/.github/workflows/package-ci.yml#L143)

Reply:
I made it so the existing tests called the examples tests, so that it automatically gets executing in the dist build